data/json/items/vehicle/controls.json

```diff
@@ -145,5 +145,21 @@
     "price": 500000,
     "price_postapoc": 3000,
     "copy-from": "vehicle_controls"
+  },
+  {
+    "type": "GENERIC",
+    "id": "vehicle_controls_xl_rig",
+    "name": { "str": "XL vehicle control rig", "str_pl": "sets of XL control rigs" },
+    "description": "A huge, adjustable combination seat and controls able to fit all sizes.",
```

Comment:
"Able to fit all sizes" is wrong, it should be restricted to huge characters.

Reply:
It is XL exclusive now.